invenio_accounts/views/rest.py

```diff
@@ -75,7 +75,7 @@ def role_to_dict(role):
     )
 
 
-def create_blueprint(app):
+def create_rest_blueprint(app):
```

Comment:
```suggestion
def create_blueprint(app):
```

nit: since we're in `rest.py` anyways and this is not exported anywhere

Reply:
my thinking was, that i don't have to add lines like
 `from invenio_accounts.views.settings import create_blueprint as create_settings_blueprint` in `conftest.py`, because  i have to import both there. And in my opinion it looks clearer in the `setup.cfg` file for the entrypoints